### ISSUE TITLE
IOS application crash in Xcode after using insomnia.keepAwake

### ISSUE THREAD

**Issue by prabh08** [nativescript-community/insomnia#11]
I am working on Nativescript 6 + Javascript application and I have used insomnia in my code but my application is getting crash due to '**insomnia**'.

Here is further error when my application is crash due to insomnia
And my IOS version is 13.1. 
> 
    2019-11-28 01:03:20.954926+0530 salesrepmobile[19942:14434690] *** Assertion failure in 
     - [FBSSerialQueue assertOnQueue], 
    /BuildRoot/Library/Caches/com.apple.xbs/Sources/FrontBoardServices_Sim/FrontBoard- 
     619.2/FrontBoardServices/FBSSerialQueue.m:98
    (lldb) 

Please help me. I am really stuck on this.
>

**Comment by EddyVerbruggen** [comment]
Is there a way for me to reproduce this perhaps? Maybe you can share a little project that reproduces it?

**Comment by prabh08** [comment]
@EddyVerbruggen Acually due to some legel boundation I can not share it. But I am very much sure that is due to insomnia.keepWake() function. Please help me. 
One more thing I want to let you know that insomnia.allowSleepAgain() is working for me but keepAwake not working.

**Comment by EddyVerbruggen** [comment]
I didn’t mean I want your project, but simply a project that demonstrates the problem.

**Comment by prabh08** [comment]
@EddyVerbruggen Can we share it on skype? I can add you on skype and we share the screen session.

**Comment by EddyVerbruggen** [comment]
No. I need code, not a demonstration. If you want me to look at it in my spare time then please make it simple for me to reproduce the issue by creating a new project with this plugin that demonstrates the problem.

**Comment by prabh08** [comment]
@EddyVerbruggen Ok I will share my code soon.
